### PR TITLE
SIG Multicluster spotlight blog

### PR DESCRIPTION
**Pull request by dewan-ahmed**
The blog resulting from this PR will serve as the canonical URL for the blog on kubernetes.io. [Details in the relevant PR](https://github.com/kubernetes/website/pull/31242) @sftim 

Once this PR is merged, I'll add a commit to [this PR](https://github.com/kubernetes/website/pull/31242) and add the canonical URL in the metadata.

### CODE REVIEW

File: content/en/blog/2022/2022-02-04-sig-multicluster-spotlight-blog.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+> I've got workloads that are spread across multiple clusters and I need them to talk to each other. 
+ 
+Okay, that's very straightforward and we know that we need to solve that. To get started, let's make sure that these projects can work together on a common API so you get the same kind of portability that you get with Kubernetes.
+
+There's a few implementations of the MCS API out there and more are being developed. But, we didn't build an implementation because depending on how you're deploying things there could be hundreds of implementations. As long as you only need the basic Multicluster service functionality, it'll just work on whatever background you want, whether it's Submariner, GKE, or a service mesh.
+
+My favortie example of "then vs. now" is cluster ID. A few years ago, there was an effort to define a cluster ID. A lot of really good thought went into this concept, for example, how do we make a cluster ID is unique across multiple clysters. How do we make this ID globally unique so it'll work in every contact? Let's say, there's an acquisition or merger of teams - does the cluster IDs still remain unique for those teams?
```
Comment (sftim) [inline content/en/blog/2022/2022-02-04-sig-multicluster-spotlight-blog.md:32]:
```suggestion
My favorite example of "then vs. now" is cluster ID. A few years ago, there was an effort to define a cluster ID. A lot of really good thought went into this concept, for example, how do we make a cluster ID is unique across multiple clusters. How do we make this ID globally unique so it'll work in every contact? Let's say, there's an acquisition or merger of teams - does the cluster IDs still remain unique for those teams?
```

File: content/en/blog/2022/2022-02-04-sig-multicluster-spotlight-blog.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+title: "Spotlight on SIG Multicluster"
+date: 2022-02-04
+slug: sig-multicluster-spotlight-2022
+---
+
+**Authors:** Dewan Ahmed (Aiven) and Chris Short (AWS)
+
+## Introduction
+
+[SIG Multicluster](https://github.com/kubernetes/community/tree/master/sig-multicluster) is the SIG focused on how Kubernetes concepts are expanded and used beyond the cluster boundary. Historically, Kubernetes resources only interacted within that boundary - KRU or Kubernetes Resource Universe (not an actual Kubernetes concept). Kubernetes clusters, even now, don't really know anything about themselves or, about other clusters. Absence of cluster identifiers is a case in point. With the growing adoption of multicloud and multicluster deployments, the work SIG Multicluster doing is gaining a lot of attention. In this blog, [Jeremy Olmsted-Thompson, Google](https://twitter.com/jeremyot) and [Chris Short, AWS](https://twitter.com/ChrisShort) discuss the interesting problems SIG Multicluster is solving and how you can get involved. Their initials **JOT** and **CS** will be used for brevity.
```
Comment (sftim) [inline content/en/blog/2022/2022-02-04-sig-multicluster-spotlight-blog.md:12]:
```suggestion
[SIG Multicluster](https://github.com/kubernetes/community/tree/master/sig-multicluster) is the Special Interest Group (SIG) focused on how Kubernetes concepts are expanded and used beyond the cluster boundary. Historically, Kubernetes resources only interacted within that boundary - KRU or Kubernetes Resource Universe (not an actual Kubernetes concept). Kubernetes clusters, even now, don't really know anything about themselves or, about other clusters. Absence of cluster identifiers is a case in point. With the growing adoption of multicloud and multicluster deployments, the work SIG Multicluster doing is gaining a lot of attention.

In this article, [Jeremy Olmsted-Thompson, Google](https://twitter.com/jeremyot) and [Chris Short, AWS](https://twitter.com/ChrisShort) discuss the interesting problems SIG Multicluster is solving and how you can get involved. Their initials **JOT** and **CS** will be used for brevity.
```

&hellip;explaining what SIG means, and also calling it an article. This is an article that appears in a blog - if you prefer using the word “blog” to mean both the set of articles and each individual article, you can omit that fix.

File: content/en/blog/2022/2022-02-04-sig-multicluster-spotlight-blog.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+**CS**: What do you think keeps people in this group? How has the pandemic affected you?
+
+**JOT**: I think it definitely got a little bit quieter during the pandemic. But for the most part; it's a very distributed group so whether you're calling in to our weekly meetings from a conference room or from your home, it doesn't make that huge of a difference. During the pandemic, a lot of people had time to focus on what's next for their scale and growth. I think that's what keeps people in the group - we have real problems that need to be solved which are very new in this space. And it's fun :)
+
+## Wrap Up
```
Comment (sftim) [inline content/en/blog/2022/2022-02-04-sig-multicluster-spotlight-blog.md:58]: style nit:
```suggestion
## Wrap up
```

**Review by sftim — CHANGES_REQUESTED** [review]
For _this_ site:

Please remove `2022-02-04-` from the Markdown file's name, so that it is:
`sig-multicluster-spotlight-blog.md`

All the inline feedback is nits (stuff that's nice to fix, but you don't have to).

**Comment by dewan-ahmed** [comment]
> For _this_ site:
> 
> Please remove `2022-02-04-` from the Markdown file's name, so that it is: `sig-multicluster-spotlight-blog.md`
> 
> All the inline feedback is nits (stuff that's nice to fix, but you don't have to).

Thanks @sftim fixed the file name and accepted suggested edits.

**Pull request merged**